src/get_objects_on_table.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python3
+import open3d as o3d
+from open3d_ros_helper import open3d_ros_helper as orh
+import copy
+import rospy
+import numpy as np
+import tf2_ros
+from tf2_sensor_msgs.tf2_sensor_msgs import do_transform_cloud
+from scipy.spatial.transform import Rotation as R
+from vision_msgs.msg import BoundingBox3D, BoundingBox3DArray
+from table_plane_extractor.srv import GetObjectsOnTable, GetObjectsOnTableResponse
+from table_plane_extractor_srv import transformPointCloud, table_plane_extractor_methode
+
+target_frame = 'base_link'
```

Comment:
should be a parameter, with a bit of documentation. It's a good default, but there's always the weird edge case where there is no base_link or base_link is not oriented with the z axis being vertical

Reply:
Implemented